backend/src/main/java/ai/verta/modeldb/config/Config.java

```diff
@@ -81,18 +82,31 @@ public FutureJdbi getJdbi() {
     if (this.jdbi == null) {
       // Initialize HikariCP and jdbi
       final var databaseConfig = config.database;
-      final var hikariDataSource = new HikariDataSource();
-      final var dbUrl = CommonHibernateUtil.buildConnectionString(databaseConfig.RdbConfiguration);
-      hikariDataSource.setJdbcUrl(dbUrl);
-      hikariDataSource.setUsername(databaseConfig.RdbConfiguration.RdbUsername);
-      hikariDataSource.setPassword(databaseConfig.RdbConfiguration.RdbPassword);
-
-      hikariDataSource.setMaximumPoolSize(10); // TODO: configure with a valid pool size
+      return initializeJdbi(databaseConfig);
+    }
+    return this.jdbi;
+  }
 
-      final Jdbi jdbi = Jdbi.create(hikariDataSource);
-      final Executor dbExecutor = FutureGrpc.initializeExecutor(databaseConfig.threadCount);
-      this.jdbi = new FutureJdbi(jdbi, dbExecutor);
+  public FutureJdbi getTestJdbi() {
+    if (this.jdbi == null) {
```

Comment:
Is it safe to use the exact same object?

Reply:
Yes. we are used same `jdbi` but used test config so yes it is safe.